samples/client/petstore/csharp/generichost/latest/Tags/src/Org.OpenAPITools/README.md

```diff
@@ -152,7 +152,7 @@ Authentication schemes defined for the API:
 - apiName: Api
 - caseInsensitiveResponseHeaders: 
 - conditionalSerialization: false
-- disallowAdditionalPropertiesIfNotPresent: 
+- disallowAdditionalPropertiesIfNotPresent: true
```

Comment:
This change in the README for one of the C# samples seems to reinforce the idea that the default value for `disallowAdditionalPropertiesIfNotPresent` is a hidden third option rather than `true`.